scripts/logid-generator.py

```diff
@@ -177,12 +180,16 @@ def main(directory, check):
   parser = OptionParser()
   parser.add_option("-d", "--directory", dest="directory", nargs=1, help="Source directory")
   parser.add_option("-c", "--check", dest="check", action="store_true", default=False, help="Check for colliding IDs")
+  parser.add_option("-e", "--excludes", dest="excludes", action="append", type=str, help="Exclude directories")
```

Comment:
Help text should mention that it is a regexp and not just directories. E.g.:

help="Exclude regex pattern"


Also the default should be an empty list to work without providing this command line argument, please test that.

default=[]